src/server/api/common/generate-replies-query.ts

```diff
@@ -1,22 +1,24 @@
 import { User } from '@/models/entities/user';
 import { Brackets, SelectQueryBuilder } from 'typeorm';
 
-export function generateRepliesQuery(q: SelectQueryBuilder<any>, me?: { id: User['id'] } | null) {
-	if (me == null) {
+export function generateRepliesQuery(q: SelectQueryBuilder<any>, me: any) {
+	const userId = me['id'] ? me['id'] : null;
+	const plTimeline: boolean = me['pleromaTimeline'] ? me['pleromaTimeline'] : false;
```

Comment:
I think using `||` or `??` (null coalescing) instead of the ternary operator might be cleaner / less duplication.

Reply:
Thank you! Sorry I'm not really a JS/TS dev, so I wasn't aware it had coalesce operators. Changed.